css/amp.css

```diff
@@ -334,3 +334,13 @@ amp-analytics {
   overflow: hidden !important;
   visibility: hidden;
 }
+
+amp-user-notification {
+  position: fixed !important;
```

Comment:
This definition ideally wouldn't be in amp.css. Can it be moved to the extension?